### PR TITLE
run pylint script

### PR DESCRIPTION
**Pull request by adrianherrera**

### CODE REVIEW

**Comment by adrianherrera** [comment]
Made some changes - instead of just providing a pylintrc file I provided a script that a user can run. This script also contains my custom string checkers.

File: pylint_rc [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+
+# Template used to display messages. This is a python new-style format string
+# used to format the message information. See doc for all details
+#msg-template=
+
+
+[FORMAT]
+
+# Maximum number of characters on a single line.
+max-line-length=80
```
Comment (vitalych) [inline pylint_rc]: 120 :)

File: run_pylint.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
```
Comment (vitalych) [inline run_pylint.py:1]: Add copyright notice

**Review by vitalych — APPROVED** [review]

**Comment by adrianherrera** [comment]
I added the copyright text. I took some code from pylint itself, which is GPL. So I've added a GPL header. If this is correct then I'll merge.

I still think the 80 char limit is the standard for Python code (PEP8, Google style guide, etc.), but I'm not fighting this battle anymore. Here's my white flag 🏳️

**Pull request merged**